ui-particles-angular/projects/ui-particles-angular/src/lib/gio-form-headers/gio-form-headers.component.ts

```diff
@@ -129,7 +132,10 @@ const HEADER_NAMES = [
 })
 export class GioFormHeadersComponent implements OnInit, ControlValueAccessor, Validator {
   @Input()
-  public adaptOutputFn?: FormHeaderAdaptOutputFn<unknown>;
+  public headerFieldMapper: FormHeaderFieldMapper = {
+    keyName: 'name',
```

Comment:
```suggestion
    keyName: 'key',
```
We must keep key here by default, otherwise we will change its behaviour wherever it is used.